### DISCUSSION TITLE
will README.md files needs to have screenshot and links

### DISCUSSION THREAD

**Discussion by charlie8658** [rohan472000/Java-DSA-InterviewPrep#8] [Q&A] [answered]
I just want to ask , for proper documentation as requested before do i need to give links and screenshots? If yes then what will be the source of links?

**Comment by rohan472000** [comment, marked as answer]
Yes that would be better to give screenshots and links, links should be from there proper websites with examples, if you are making docs for sorting algos then you can give links like https://www.geeksforgeeks.org/sorting-algorithms/

**Reply by charlie8658** [reply]
thanks.
